scripts/vcf2hdf5_parallel

```diff
@@ -253,6 +253,17 @@ def Main(argv=None):
                     chrom = fields[args.regions_column].split(':')[0]
                     if chrom not in chrom_list:
                         chrom_list[chrom] = True
+
+    # check chrom_list for invalid chars '_' and ':'
+    # @TCC at least '_' could probably be allowed if vcf2npy output file nameing was changed (need a separator between chrom and position range)
+    for c in chrom_list:
+        if '_' in c:
+            logging.critical("character '_' in chrom name '{}' will probably break this script".format(c))
+            sys.exit(2)
+        if ':' in c:
+            logging.critical("character ':' in chrom name '{}' will probably break this script".format(c))
+            sys.exit(2)
+             
```

Comment:
The better solution is to modify `vcf2npy` so that it outputs filenames of the format `{array_type}.{chrom}:{start}-{stop}` as opposed to the current `{array_type}.{chrom}_{start}_{stop}`.  Then '_' would be fine.  

The difficulty is that it may break some of your workflow/scripts which use `vcf2npy`.  

Another alternative is to actually just check for the problematic case where there is `{A}` and `{A}_*` in the `chrom_list`.  I can't think of a particularly clever way of doing that, but since it only needs to check names containing an '_', brute force is probably ok.